media-sound/mac/mac-4.38.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A fast and powerful lossless audio compressor"
+DESCRIPTION="Monkey's Audio is a lossless audio format. This package provides \
+the mac command line utility for compressing and decompressing Monkey's Audio \
+files."
+HOMEPAGE="https://www.monkeysaudio.com/"
+COPYRIGHT="2000-2018 Matthew T. Ashland"
+LICENSE="Monkey's Audio SDK and Source Code License Agreement"
+REVISION="1"
+SOURCE_URI="https://www.monkeysaudio.com/files/MAC_SDK_438.zip"
+CHECKSUM_SHA256="1ab9a0c74a0478c3fe713908789017c53c3b333a482225d2d09990a16069f981"
+SOURCE_DIR=""
+PATCHES="mac-$portVersion.patchset"
+ADDITIONAL_FILES="Makefile"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
```

Comment:
Here, since the package provides a shared lib, we should make x86 secondary arch available no matter what the primary arch is.
(Yeah, there aren't official images of x86_64/x86 hybrid yet, but this will probably happen at some point.)

```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
-if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
-fi
 
+commandSuffix=$secondaryArchSuffix
 commandBinDir=$binDir
 if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
 	commandBinDir=$prefix/bin
 fi
```